CoThings/ViewController.swift

```diff
@@ -332,3 +325,46 @@ class ViewController: UIViewController, CLLocationManagerDelegate, UITableViewDe
         print("You tapped cell number \(indexPath.row).")
     }
 }
+
+
+
+class MessageAdapter {
```

Comment:
This class sounds like it's better to be in a separate file. At least not in ViewController.

Reply:
yeah I can do that

Reply:
Thanks a lot!